src/css/main.less

```diff
@@ -838,36 +838,25 @@ fieldset[disabled] .btn-info.active {
   background-color: #5e646b;
 }
 
-.health-dot {
-  display: inline-block;
-}
 
-.health-dot:before {
-  content: "\00a0";
-  border-radius: 50%;
-  border: solid 2px #51565d;
-  width: 12px;
-  height: 12px;
-  display: inline-block;
-  vertical-align: bottom;
-  margin-right: 6px;
-  margin-bottom: 1px;
-}
+/* ==============
```

Comment:
I don't see comments like that in the main.less, that would be the only one, but fair enough.

Little note:
Let's do things only at the point we need them.
Don't think to the future in terms of preparing code beforehand, you never know how the future will be. That could let to unnecessary work and if it's not needed also unclear.
Architecturally you should have something in mind of course.